queue/queue.go

```diff
@@ -22,6 +25,18 @@ const (
 
 type Store struct {
 	*bolt.DB
+	pubsub.Publisher
+}
+
+func (db *Store) publishResponseJSON(resp mdm.Response) error {
+	buf := new(bytes.Buffer)
+	enc := json.NewEncoder(buf)
```

Comment:
I wanted to have a bit of API parity bewtween the command submit API and the command receive API. I.e. both of them JSON. The mdm.Response also contains other data that might be useful to processing a command (or at least for debugging).

I considered making a new /api/v1/rawcommand/<uuid> API where you submit actual raw Plist MDM commands that go directly on the queue (and you would indeed get raw Plist commands coming from the client on the callback).

Didn't go that way honestly because this way is much, much easier. :)